resources/com.sap.piper/templates/mvn_download_cache_proxy_settings.xml

```diff
@@ -0,0 +1,10 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+    <mirrors>
+        <mirror>
+            <id>s4sdk-mirror</id>
```

Comment:
You could rename the mirror, e.g. download-cache

Reply:
done